### ISSUE TITLE
Tokenisation issues in CoNLL-U format description

### ISSUE THREAD

**Issue by jeanm** [UniversalDependencies/docs#322]
The [CoNLL-U format description](universaldependencies.org/format.html) has several examples in English which contain contractions, presumably to illustrate how multiword tokens should be annotated. One example:

```
1     I         I
2-3   haven't   _
2     have      have
3     not       not
```

These examples might be a bit confusing/misleading, because this is not how English text is normally tokenised. Even the English Web Treebank, which is the source of the `UD_English` corpus, would not tokenise like that. Most tokenisers/corpora would instead have:

```
1     I         I
2-3   haven't   _
2     have      have
3     n't       not
```

If the maintainers agree, I am happy to submit a PR with the 'standard' tokenisation.

**Comment by dan-zeman** [comment]
This issue is closely related to https://github.com/UniversalDependencies/UD_English/issues/5. Whatever the final solution there (and in the data), the examples in documentation should follow.

**Comment by arademaker** [comment]
It seems people are not using the suggested format for annotating multiword. 

> To accommodate the representation of both words and (multiword) tokens (see Tokenization), we adopt an extension of the original CoNLL-X token indexing scheme, where words are indexed with integers 1, 2, 3, …, while (multiword) tokens are indexed with integer ranges like 1-2 or 3-5. Lines representing such tokens are inserted before the first word in the range. These ranges must be nonempty and must not overlap. They have a FORM value – the string that occurs in the sentence – but have an underscore in all the remaining fields (because the token represents multiple words, each with its own lemma, part-of-speech tag, syntactic head, and so on). 

http://universaldependencies.org/format.html

In the English corpus https://raw.githubusercontent.com/UniversalDependencies/UD_English/master/en-ud-test.conllu I found examples of contractions without these lines with the range of tokens.  Is it true? Should be ignore these explicit annotation of contractions?

**Comment by dan-zeman** [comment]
Perhaps it would be better to use examples from a language other than English in the universal guidelines. The exact set of contractions is specific to each language but if we cannot find a consensus about English then we should move to a language where the data do not deviate from the guidelines.

The version 2 of the guidelines is being prepared right now. I am leaving this issue here as a reminder that we should check the examples in format.html too.

**Comment by spyysalo** [comment]
@dan-zeman : gentle reminder that now would be a very good time if you're willing to address this issue before the v2 release :-)

**Comment by dan-zeman** [comment]
Hopefully fixed in 9a98cb9e66159be77cdf00393afd60bc01645d78.
